solr/solrj/src/test/org/apache/solr/client/solrj/embedded/SolrExampleJettyTest.java

```diff
@@ -72,15 +65,15 @@ public void testBadSetup() {
 
   @Test
   public void testArbitraryJsonIndexing() throws Exception {
-    HttpSolrClient client = (HttpSolrClient) getSolrClient();
+    SolrClient client = getSolrClient();
```

Comment:
should be closed eventually?

Reply:
these get closed by the base test case...    However, happy to wrap it in a try with....    Maybe we should remove the magic closing in the base test case?

Reply:
ah ok nvm then. guess this just returns a reference to a solr client.